### PR TITLE
add prevent-default options

### PR DESCRIPTION
**Pull request by vuquangpham**
I have the option to turn on/off the `prevent-default` option when clicking on trigger elements

### CODE REVIEW

File: dev/style.scss [hunk 1]
```diff
@@ -54,6 +54,9 @@ img {
   margin-bottom: 20px;
 }
 
+[data-eta-trigger] {
```
Comment (phucbm) [inline dev/style.scss:57]: Why do we need this? And is this related to "add prevent-default option"?

File: dev/index.html [hunk 2]
```diff
@@ -199,7 +199,7 @@ <h2 class="heading-separator"><span>Accordion initialize with JSON format</span>
 
         <!-- section 2 -->
         <div>
-            <button data-eta-trigger="section-2">Section 2</button>
+            <a href="https://google.com" data-eta-trigger="section-2">Section 2</a>
```
Comment (phucbm) [inline dev/index.html:202]: Should these links open in new tab?

File: src/_index.js [hunk 3]
```diff
@@ -24,7 +24,8 @@ export class EasyTabAccordion{
             receiver: 'data-eta-receiver',
             hash: 'data-eta-hash',
             hashScroll: 'data-eta-hash-scroll',
-            animation: 'data-eta-animation'
+            animation: 'data-eta-animation',
+            preventDefault: 'data-eta-prevent-default',
```
Comment (phucbm) [inline src/_index.js:28]: We will not create more attributes for the new options as we prefer settings via JSON instead.

**Review by phucbm — CHANGES_REQUESTED** [review]
Please check my comments.

File: dev/style.scss (hunk 1 above)
Comment (vuquangpham) [reply dev/style.scss:57]: I just want to make the permalink be `block` (to have a margin-bottom CSS property)

**Review by phucbm — APPROVED** [review]
LGTM.
